activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add `values_at` method.
```

Comment:
Can you expand this CHANGELOG entry to tell where this method is and to show an example?

Reply:
Sure ! I will do it tonight

Reply:
Done. I can change it if needed.